mubench.pipeline/tests/data/test_detector.py

```diff
@@ -71,6 +71,19 @@ def test_raises_on_no_matching_release(self):
         self.setup_releases([{"md5": "-md5-", "tag": "-release-", "cli_version": "0.0.1"}])
         assert_raises(ValueError, Detector, self.temp_dir, self.detector_id, [], "-unavailable_release-")
 
+    def test_release_is_case_insensitive(self):
```

Comment:
Please make the differences in casing more obvious, like define 'TAG', request 'TaG', URL 'tag'. Also drop the '-' around the tag, because our convention is that these express "this string is an arbitrary-but-fixed value" where in this case is actually is not.

The 'cli_version' on the other hand, could be an arbitrary string here.